src/test/java/Class3/Homework3.java

```diff
@@ -0,0 +1,100 @@
+package Class3;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework3 {
+    @Test
+    public void feelsLikeTemp() {
+       System.setProperty("webdriver.chrome.driver", "./drivers/chromedriver.exe");
+
+       WebDriver driver = new ChromeDriver();
+       driver.get("https://darksky.net/forecast/40.7127,-74.0059/us12/en");
+
+        WebElement flTempValue = driver.findElement(By.className("feels-like-text"));
+        String flTempValueWithDegree = flTempValue.getText();
+        String flTempValueNoDegree = flTempValueWithDegree.substring(0, flTempValueWithDegree.length()-1);
+        int flTemp = Integer.parseInt(flTempValueNoDegree);
+
+        WebElement lowTempValue = driver.findElement(By.className("low-temp-text"));
+        String lowTempValueWithDegree = lowTempValue.getText();
+        String lowTempValueNoDegree = lowTempValueWithDegree.substring(0, lowTempValueWithDegree.length()-1);
+        int lowTemp = Integer.parseInt(lowTempValueNoDegree);
+
+        WebElement highTempValue = driver.findElement(By.className("high-temp-text"));
+        String highTempValueWithDegree = highTempValue.getText();
+        String highTempValueNoDegree = highTempValueWithDegree.substring(0, highTempValueWithDegree.length()-1);
+        int highTemp = Integer.parseInt(highTempValueNoDegree);
+
+        if (flTemp >= lowTemp && flTemp <= highTemp) {
```

Comment:
Instead if-else use Assert to verify the testcase

Reply:
boolean result = flTemp >= lowTemp && flTemp <= highTemp;
Assert.assertTrue(  result , "Feels like temp is less or greater than low-high temp"   ) ;